plugins/node/readme.md

```diff
@@ -0,0 +1,34 @@
+# @dotcom-tool-kit/node
+
+Run your Node application.
+
+This plugin will be installed as a dependency of the [frontend-app](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/frontend-app), [backend-app](https://github.com/Financial-Times/dotcom-tool-kit/tree/main/plugins/backend-app)  plugins so you do not need to install it separately if you are using either of those plugins.
```

Comment:
**praise:** This is a great addition to let teams know they might already have it installed for their app 🙌